app/services/tts_generator.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+require "google/cloud/text_to_speech"
+require "yaml"
+require "stringio"
+
+## TODO: for queue job
```

Comment:
Can we remove that `TODO`?

Reply:
Oh yes, this was just for me, forgot to remove it 🤦‍♂️